core/import-shearwater.c

```diff
@@ -483,7 +483,7 @@ int parse_shearwater_cloud_buffer(sqlite3 *handle, const char *url, const char *
 	state.target_table = table;
 	state.sql_handle = handle;
 
-	char get_dives[] = "select l.number,startTimestamp,location||' / '||site,buddy,notes,imperialUnits,maxDepth,maxTime,startSurfacePressure,computerSerial,computerModel,d.diveId,l.sampleRateMs FROM dive_details AS d JOIN dive_logs AS l ON d.diveId=l.diveId";
```

Comment:
Just to clarify, @mturkia was writing this PR to help me import my Shearwater Cloud database.  I dive with 2 dive computers, and one had suffered significant clock drift.  In the end, the `DiveDate`s all looked good, but I just wanted to clarify that the `startTimestamp` might not have been the culprit and it might have been my DC. If no problem ever comes up, `DiveDate` is probably the correct option anyways.